net.py

```diff
@@ -1,172 +0,0 @@
-import numpy as np
-
```

Comment:
Read more about Docstrings https://www.python.org/dev/peps/pep-0257 . Explain what this module do (follow PEP recomendation). It's very useful, later you can generate documentation.